ostia-operator/Makefile

```diff
@@ -2,6 +2,7 @@ NAME=ostia-operator
 NAMESPACE=quay.io/3scale
```

Comment:
maybe a `deploy` target would be beneficial here with a small docs update? Most pieces for a deployment are in place with this pr.